CHANGELOG.md

```diff
@@ -1,20 +1,23 @@
 Changelog
 =========
-## 1.3.0 (2020-xx-xx)
+## 1.3.0 (2021-01-03)
+* BC Break: dropped support for Symfony `<4.4`,
+* BC Break: dropped support for Doctrine Bundle `<2.0`,
 * Added PHP 8 support,
-* Method `ConnectController::getParameter()` has been removed,
```

Comment:
@XWB That method was removed from our controller, but after deps bump we use the one from parent class:
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L65